src/components/Autocomplete/components/ComboBox/tests/ComboBox.test.tsx

```diff
@@ -50,16 +52,16 @@ describe('<ComboBox/>', () => {
     },
   ];
 
+  const mockProps = {
+    options,
+    selected: [],
+    textField: renderTextField(),
+    onSelect: noop,
+  };
+
   describe('options', () => {
     it('passes options to OptionList', () => {
-      const comboBox = mountWithAppProvider(
-        <ComboBox
-          options={options}
-          selected={[]}
-          textField={renderTextField()}
-          onSelect={noop}
-        />,
-      );
+      const comboBox = mountWithAppProvider(<ComboBox {...mockProps} />);
 
       comboBox.simulate('click');
 
```

Comment:
Instead of this
```ts
expect(optionListOptions[0].value).toBe('cheese_pizza');
expect(optionListOptions[0].label).toBe('Cheese Pizza');
expect(optionListOptions[1].value).toBe('macaroni_pizza');
expect(optionListOptions[1].label).toBe('Macaroni Pizza');
```

can't you do a simple assertion on the prop?

```ts
expect(comboBox.find(OptionList).prop('options')).toEqual(expectedOptions)
```